Pipfile

```diff
@@ -6,23 +6,23 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-certifi = "==2020.6.20"
-cffi = "==1.14.2"
-chardet = "==3.0.4"
-colorama = "==0.4.3"
-cryptography = "==3.3.2"
-idna = "==2.10"
-iso8601 = "==0.1.12"
-pycparser = "==2.20"
-python-dateutil = "==2.8.1"
-requests = "==2.24.0"
-securesystemslib = "==0.20.1"
-six = "==1.15.0"
+certifi = "*"
+cffi = "*"
+chardet = "*"
+colorama = "*"
+cryptography = "*"
+idna = "*"
+iso8601 = "*"
+pycparser = "*"
+python-dateutil = "*"
+requests = "*"
+securesystemslib = "*"
+six = "*"
 tuf = "==0.19.0"
-urllib3 = "==1.25.10"
-PyNaCl = "==1.4.0"
+urllib3 = "*"
+PyNaCl = "*"
 flake8 = "*"
 dirhash = "*"
 
 [requires]
-python_version = "3.9"
+python_version = "3.8"
```

Comment:
This change is not simply a weakening but a re-pinning on an older Python release. We could consider dropping this entirely if it needs to be more flexible, as `Pipfile` does not allow a more flexible (but specified) constraint. Dependencies will impose implicit constraints, anyway. The Pipenv developers have [defended this behavior as a best practice](https://github.com/pypa/pipenv/issues/2683).

Reply:
Reverted.